config/prometheus/monitor.yaml

```diff
@@ -11,6 +10,10 @@ spec:
   endpoints:
     - path: /metrics
       port: https
```

Comment:
What abput doing all needed stuff to dont have proxy enabled, and have a servicemonitor pointing to a condifured http port on the operator deployment?

Reply:
out of the scope of the PR ;)

Reply:
Ok, fair enough!

I commented it because I saw that you already commented a few lines of the ServiceMonitor related to that:  `scheme`, `bearerTokenFile` and `tlsConfig`, so I thought you intention was to disable the proxy